### DISCUSSION TITLE
请问笔记发图片会显示错误，后台发表文章媒体都没有问题。

### DISCUSSION THREAD

**Discussion by KEZIYs** [Tokinx/Wing#16] [问答]
![屏幕截图 2022-06-01 141906](https://user-images.githubusercontent.com/80903988/171340418-83cd6004-2eaf-42ac-817a-ef4684b87fe0.jpg)
主题版本：0.7.7
错误显示：
`Cannot read properties of undefined (reading 'get')`
应该如何解决呢，小白一个，但很喜欢这个主题…不好意思麻烦了

**Comment by KEZIYs** [comment]
这个问题好像是，图片不能带中文吧…就很无语哈哈哈哈

**Reply by Tokinx** [reply]
我回去试一下哈

**Reply by Tokinx** [reply]
这个应该不是主题限制的，因为上传文件调用的是WP的API，或许你需要排查一下主机的配置

**Comment by guojian1993** [comment]
我遇到了类似的问题，发表微博的额时候不能上传图片

**Comment by guojian1993** [comment]
好的，我也是小白，不太懂还是谢谢回复，我自己再琢磨琢磨
用了你的主题很喜欢，下午随手打赏了下
以后可以考虑出收费版的，还是有很多人愿意为好作品买单
祝网站越办越好~




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年8月3日(星期三) 下午5:29
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [Tokinx/Wing] 请问笔记发图片会显示错误，后台发表文章媒体都没有问题。 (Discussion #16)





 
这个应该不是主题限制的，因为上传文件调用的是WP的API，或许你需要排查一下主机的配置
